### PR TITLE
fix(api): rename buildinfo and release name

### PR DESCRIPTION
**Pull request by sguiheux**
1. Description
1. Related issues
1. About tests
1. Mentions

@ovh/cds

### CODE REVIEW

File: contrib/grpcplugins/action/artifactory-plugins-lib/artifactory-plugins-lib.go [hunk 1]
```diff
@@ -323,10 +323,10 @@ func ReleaseArtifactoryRunResult(ctx context.Context, c *actionplugin.Common, re
 	return nil
 }
 
-func createReleaseBundle(ctx context.Context, c *actionplugin.Common, distriClient art.DistribClient, projectKey, workflowName, version, buildInfo string, artifactPromoted []string, destMaturity, releaseNote string) (string, string, error) {
-	buildInfoName := fmt.Sprintf("%s/%s/%s", buildInfo, projectKey, workflowName)
+func createReleaseBundle(ctx context.Context, c *actionplugin.Common, distriClient art.DistribClient, projectKey, vcs, repo, workflowName, version, buildInfo string, artifactPromoted []string, destMaturity, releaseNote string) (string, string, error) {
+	buildInfoName := fmt.Sprintf("%s/%s/%s/%s/%s", buildInfo, projectKey, vcs, repo, workflowName)
 
-	params := services.NewCreateReleaseBundleParams(strings.Replace(buildInfoName, "/", "-", -1), version)
+	params := services.NewCreateReleaseBundleParams(strings.Replace(buildInfoName, "/", "-", -1)[0:120], version)
```
Comment (sguiheux) [inline contrib/grpcplugins/action/artifactory-plugins-lib/artifactory-plugins-lib.go:329]: release name is limited to 120 characters

File: contrib/grpcplugins/action/debianPush/main.go [hunk 2]
```diff
@@ -210,11 +210,9 @@ func (p *debianPushPlugin) UploadArtifactoryDebianPackage(ctx context.Context, o
 
 	debInfo := fmt.Sprintf("%s;%s;%s", distribLayout, componentLayout, archLayout)
 	cdsInfo := fmt.Sprintf("cds_version=%s;cds_workflow=%s", opts.jobContext.Git.SemverCurrent, opts.jobContext.CDS.Workflow)
-	buildInfo := fmt.Sprintf("build.name=%s/%s/%s;build.number=%d;build.timestamp=%d",
-		opts.jobContext.Integrations.ArtifactManager.Get(sdk.ArtifactoryConfigBuildInfoPrefix), opts.jobContext.CDS.ProjectKey, opts.jobContext.CDS.Workflow, opts.jobContext.CDS.RunNumber, time.Now().Unix())
 	maturity := opts.jobContext.Integrations.ArtifactManager.Get(sdk.ArtifactoryConfigPromotionLowMaturity)
-	path := fmt.Sprintf("/pool/%s;%s;%s;%s;deb.release.origin=%s;deb.release.label=%s",
-		fileName, debInfo, cdsInfo, buildInfo, opts.origin, opts.label)
+	path := fmt.Sprintf("/pool/%s;%s;%s;deb.release.origin=%s;deb.release.label=%s",
```
Comment (sguiheux) [inline contrib/grpcplugins/action/debianPush/main.go:214]: I removed buildinfo as it's set by api

**Comment by ovh-cds** [comment]
CDS Report build-cds#21060.0 ✔ 
* 
  * Build and Package All -   
  * Build and Package Contrib Action ✔ 
  * Build and Package Contrib Code Plugin ✔ 
  * Build and Package Contrib Integrations ✔ 
  * Build and Package Engine, Worker, CLI and Doc ✔ 
  * Build and Package UI ✔ 
  * Build Debpacker ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Contrib ✔ 
  * Test Engine ✔ 
  * Test SDK ✔ 
  * Test UI ✔

**Comment by ovh-cds** [comment]
CDS Report integration-cds#21060.0 ✘  
* Integration
  * Docker Compose Tests ✔ 
  * Standalone Tests ✔ 
  * Standalone Tests v1 ✘  
  * Standalone Tests v2 ✘

**Comment by ovh-cds** [comment]
CDS Report integration-cds#21060.1 ✘  
* Integration
  * Docker Compose Tests ✔ 
  * Standalone Tests ✔ 
  * Standalone Tests v1 ✔ 
  * Standalone Tests v2 ✘

**Comment by ovh-cds** [comment]
CDS Report build-cds#21064.0 ✔ 
* 
  * Build and Package All -   
  * Build and Package Contrib Action ✔ 
  * Build and Package Contrib Code Plugin ✔ 
  * Build and Package Contrib Integrations ✔ 
  * Build and Package Engine, Worker, CLI and Doc ✔ 
  * Build and Package UI ✔ 
  * Build Debpacker ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Contrib ✔ 
  * Test Engine ✔ 
  * Test SDK ✔ 
  * Test UI ✔

**Comment by ovh-cds** [comment]
CDS Report build-cds#21066.0 ✔ 
* 
  * Build and Package All -   
  * Build and Package Contrib Action ✔ 
  * Build and Package Contrib Code Plugin ✔ 
  * Build and Package Contrib Integrations ✔ 
  * Build and Package Engine, Worker, CLI and Doc ✔ 
  * Build and Package UI ✔ 
  * Build Debpacker ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Contrib ✔ 
  * Test Engine ✔ 
  * Test SDK ✔ 
  * Test UI ✔

**Comment by ovh-cds** [comment]
CDS Report integration-cds#21066.0 ✔ 
* Integration
  * Docker Compose Tests ✔ 
  * Standalone Tests ✔ 
  * Standalone Tests v1 ✔ 
  * Standalone Tests v2 ✔

**Pull request merged**